cves/kernel/CVE-2012-6545.yml

```diff
@@ -319,9 +324,10 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    The fix went through quite silently.
```

Comment:
You mentioned there was correspondence between two people under your "Vouch" notes.  Would this not qualify as a discussion, even security-related as they were discussing the potential of a CVE?

Reply:
I could mark it down for that just to have that discussion linked, could be important to someone.